content/en/blog/_posts/get-started-with-kubernetes-using-python.md

```diff
@@ -0,0 +1,217 @@
+
+```
+docker image ls
+```
+<img src="/images/blog/get-started-with-kubernetes-using-python/docker-image-ls.png" alt="Docker image listing" />
+
+The application is now containerized, which means it can now run in Docker and Kubernetes!
+
+## Running in Docker
+Before jumping into Kubernetes, let’s verify it works in Docker.
+Run the following command to have Docker run the application in a container and map it to port 5001:
+```
+docker run -p 5001:5000 hello-python
+```
+Now navigate to http://localhost:5001, and you should see the “Hello form Python!” message. 
+
+### More info
+* [Get started with Docker](https://docs.docker.com/get-started/)
+* [Best practices for writing Dockerfiles](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/)
+* [Docker Cheat Sheet](https://www.docker.com/sites/default/files/Docker_CheatSheet_08.09.2016_0.pdf) (pdf)
+
+## Running in Kubernetes
+You are finally ready to get the application running in Kubernetes. Because you have a web application, you will create a service and a deployment.
+
+First verify your kubectl is configured. At the command line, type the following:
+```
+kubectl version
+```
+
+If you don’t see a reply with a Client and Server version, you’ll need to [install](https://kubernetes.io/docs/tasks/tools/install-kubectl/) and configure it.
+
+Make sure it is using the Docker for Desktop context by running the following:
+```
+kubectl config use-context docker-for-desktop
+```
+
+Now you are working with Kubernetes! You can see the node by typing: 
+```
+kubectl get nodes
+```
+
+Now let’s have it run the application. Create a file named deployment.yaml and add the following contents to it and then save it:
+
+```
+apiVersion: v1
+kind: Service
+metadata:
+  name: hello-python-service
+spec:
+  selector:
+    app: hello-python
+  ports:
+  - protocol: "TCP"
+    port: 6000
+    targetPort: 5000
+  type: LoadBalancer
+
+---
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: hello-python
+spec:
+  replicas: 4
+  template:
+    metadata:
+      labels:
+        app: hello-python
+    spec:
+      containers:
+      - name: hello-python
+        image: hello-python:latest
+        imagePullPolicy: Never
+        ports:
+        - containerPort: 5000
+```
+
+This YAML file is the instructions to Kubernetes for what you want running. It is telling Kubernetes the following:
+* You want a load-balanced service exposing port 6000
+* You want four instances of the hello-python container running 
+
+Use kubectl to send the YAML file to Kubernetes by running the following command:
+```
+kubectl apply -f deployment.yaml
+```
+
+You can see the pods are running if you execute the following command:
+```
+kubectl get pods
+```
+<img src="/images/blog/get-started-with-kubernetes-using-python/kubectl-get-pods.png" alt="Pod listing" />
+
+Now navigate to http://localhost:6001, and you should see the “Hello form Python!” message. 
```

Comment:
The load balancer is exposed on port 6000 in the configuration above, not 6001.

Reply:
Thanks, nice catch!